lib/elixir/lib/float.ex

```diff
@@ -46,6 +46,41 @@ defmodule Float do
   @precision_range 0..15
   @type precision_range :: 0..15
 
+  @doc """
+  Computes `base` raised to power of `exponent`.
+
+  `base` must be a float and `exponent` can be any number.
+  However, if a negative base and a fractional exponent
+  are given, it raises argument error.
```

Comment:
Do we use "argument error" to refer all argument errors? Or only `ArgumentError`?  I found it is captured as `ArithmeticError` in Elixir 1.1.2 / OTP 23.1.2

```elixir
defmodule Foo do
  def pow(base, exponent) when is_float(base) and is_number(exponent),
    do: :math.pow(base, exponent)
end

Foo.pow(-1.0, 1.1)
** (ArithmeticError) bad argument in arithmetic expression
    (stdlib 3.13.2) :math.pow(-1.0, 1.1)
    iex:3: Foo.pow/2
```

Reply:
```suggestion
  are given, it raises `ArithmeticError`.
```